File: Android Java Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/oms/OrphanageActivities/OrphanageActivities.java
package com.kaligotla.oms.OrphanageActivities;

import com.kaligotla.oms.orphanage.Orphanage;

public class OrphanageActivities {
    private int event_id;

    private Orphanage orphanage;

    private String details, image_1, image_2, image_3, image_4, image_5;

    public OrphanageActivities() {
    }

    public OrphanageActivities(Orphanage orphanage, String details, String image_1, String image_2, String image_3, String image_4, String image_5) {
        this.orphanage = orphanage;
        this.details = details;
        this.image_1 = image_1;
        this.image_2 = image_2;
        this.image_3 = image_3;
        this.image_4 = image_4;
        this.image_5 = image_5;
    }

    public int getEvent_id() {
        return event_id;
    }

    public void setEvent_id(int event_id) {
        this.event_id = event_id;
    }

    public Orphanage getOrphanage() {
        return orphanage;
    }

    public void setOrphanage(Orphanage orphanage) {
        this.orphanage = orphanage;
    }

    public String getDetails() {
        return details;
    }

    public void setDetails(String details) {
        this.details = details;
    }

    public String getImage_1() {
        return image_1;
    }

    public void setImage_1(String image_1) {
        this.image_1 = image_1;
    }

    public String getImage_2() {
        return image_2;
    }

    public void setImage_2(String image_2) {
        this.image_2 = image_2;
    }

    public String getImage_3() {
        return image_3;
    }

    public void setImage_3(String image_3) {
        this.image_3 = image_3;
    }

    public String getImage_4() {
        return image_4;
    }

    public void setImage_4(String image_4) {
        this.image_4 = image_4;
    }

    public String getImage_5() {
        return image_5;
    }

    public void setImage_5(String image_5) {
        this.image_5 = image_5;
    }

    @Override
    public String toString() {
        return "OrphanageActivities{" +
                "orphanage_id=" + orphanage.getId() +
                ", details='" + details + '\'' +
                ", image_1='" + image_1 + '\'' +
                ", image_2='" + image_2 + '\'' +
                ", image_3='" + image_3 + '\'' +
                ", image_4='" + image_4 + '\'' +
                ", image_5='" + image_5 + '\'' +
                '}';
    }
}

File: Flutter Dart Node Express - Orphanslife/lib/Models/orphanage_activities.dart
import 'dart:convert';

List<OrphanageActivities> modelUserFromJson(String str) =>
    List<OrphanageActivities>.from(json.decode(str).map((x) => OrphanageActivities.fromJson(x)));

class OrphanageActivities {
  String _event_id, _orphanage_id, _details, _image_1, _image_2, _image_3, _image_4, _image_5;

  OrphanageActivities.all(this._event_id,this._orphanage_id,this._details,this._image_1,this._image_2,this._image_3,this._image_4,this._image_5);

  factory OrphanageActivities.fromJson(Map<String, dynamic> json) => OrphanageActivities.all(
      json["even_id"].toString(),
      json["orphanage_id"].toString(),
      json["details"].toString(),
      json["image_1"].toString(),
      json["image_2"].toString(),
      json["image_3"].toString(),
      json["image_4"].toString(),
      json["image_5"].toString()
  );

  Map<String, dynamic> toJson() => {
    "id": event_id,
    "amount": orphanage_id,
    "payment_status": details,
    "image_1" : image_1,
    "image_2" : image_2,
    "image_3" : image_3,
    "image_4" : image_4,
    "image_5" : image_5
  };

  get image_5 => _image_5;

  set image_5(value) {
    _image_5 = value;
  }

  get image_4 => _image_4;

  set image_4(value) {
    _image_4 = value;
  }

  get image_3 => _image_3;

  set image_3(value) {
    _image_3 = value;
  }

  get image_2 => _image_2;

  set image_2(value) {
    _image_2 = value;
  }

  get image_1 => _image_1;

  set image_1(value) {
    _image_1 = value;
  }

  get details => _details;

  set details(value) {
    _details = value;
  }

  get orphanage_id => _orphanage_id;

  set orphanage_id(value) {
    _orphanage_id = value;
  }

  String get event_id => _event_id;

  set event_id(String value) {
    _event_id = value;
  }

  @override
  String toString() {
    return 'OrphanageActivities{event_id: $_event_id, orphanage_id: $_orphanage_id, details: $_details, image_1: $_image_1,'
        ' image_2: $_image_2, image_3: $_image_3, image_4: $_image_4, image_5: $_image_5}';
  }
}